pyproject.toml

```
@@ -41,3 +41,7 @@ platforms = ["Windows", "Linux", "Mac OS-X"]
 
 [tool.setuptools.packages.find]
 where = ["."]
+
+[tool.ruff]
+target-version = "py38"
```

Same as the other comment.

```suggestion
target-version = "py39"
```

---

Done